blocks/default-output-block/output-types/__tests__/default.test.jsx

```diff
@@ -528,3 +529,23 @@ describe('queryly render conditions', () => {
     expect(wrapper.find('script').find({ 'data-integration': 'queryly' }).length).toBe(2);
   });
 });
+
+describe('The spa property', () => {
+  // we're only expecting spaSites to be an array
```

Comment:
Is this comment correct, as below we have a test for passing in boolean values

Reply:
ah I thought you wanted the behavior shown as if the bools were passed in. I can remove the boolean tests

Reply:
It's a little confusing the way it reads and the code is testing - if ThemeSettings decides the property could be a array or boolean, that's up to them.

Reply:
updated